libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -25,6 +26,16 @@ type Metadata = {
   cover_art_sizes: string
 }
 
+type PlaylistMetadata = {
```

Comment:
there's no `action` field here, so would remove from above

Reply:
there's also no entityType field on this, but don't think it needs it

Reply:
totally agreed